opencti-platform/opencti-graphql/src/modules/internal/document/document.ts

```diff
@@ -17,7 +18,7 @@ const attributes: Array<AttributeDefinition> = [
   { name: 'information', label: 'Information', type: 'string', format: 'short', mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: false },
   { name: 'lastModified', label: 'Last modification date', type: 'date', mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: true },
   { name: 'lastModifiedSinceMin', label: 'Last modification since', type: 'numeric', precision: 'integer', mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: false },
-  { name: 'uploadStatus', label: 'Upload status', type: 'string', format: 'enum', values: ['wait', 'progress', 'complete', 'timeout'], mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: true },
+  { name: 'uploadStatus', label: 'Upload status', type: 'string', format: 'enum', values: Object.values(State), mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: true },
```

Comment:
for consistency, could you please put it as a constant above ? like in other files.

Reply:
fixed